pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -291,7 +306,13 @@ pub mod pallet {
 		/// A season's treasury has been claimed by a treasurer.
 		TreasuryClaimed { season_id: SeasonId, treasurer: T::AccountId, amount: BalanceOf<T> },
 		/// The season configuration for {season_id} has been updated.
-		UpdatedSeason { season_id: SeasonId, season: SeasonOf<T> },
+		UpdatedSeason {
+			season_id: SeasonId,
+			season: Option<SeasonOf<T>>,
+			meta: Option<SeasonMeta>,
+			schedule: Option<SeasonScheduleOf<T>>,
+			trade_filters: Option<TradeFilters>,
+		},
```

Comment:
Hmm, this is interesting, from a client perspective, I wonder if it is more convenient to have separate events? In this case, you can only subscribe to the relevant event, and you don't need extra logic to find out what exactly has changed. Even more so, I guess there are fields that we never really expect to change?